models/src/it/java/io/sphere/sdk/products/ProductProjectionSearchIntegrationTest.java

```diff
@@ -46,10 +46,10 @@
     @BeforeClass
     public static void setupProducts() {
         removeProducts();
-        final TextAttributeDefinition colorAttributeDefinition = TextAttributeDefinitionBuilder
-                .of(COLOR, LocalizedStrings.ofEnglishLocale(COLOR), TextInputHint.SINGLE_LINE).build();
-        final TextAttributeDefinition sizeAttributeDefinition = TextAttributeDefinitionBuilder
-                .of(SIZE, LocalizedStrings.ofEnglishLocale(SIZE), TextInputHint.SINGLE_LINE).build();
+        final AttributeDefinition colorAttributeDefinition = AttributeDefinitionBuilder
+                .of(COLOR, LocalizedStrings.ofEnglishLocale(COLOR), new TextType()).build();
```

Comment:
I think you merged master and there is the new naming applied.

The `of` instead of `new` should be kind of everywhere in public APIs.